planning/behavior_path_planner/include/behavior_path_planner/utils/utils.hpp

```diff
@@ -323,8 +323,8 @@ PathPointWithLaneId insertStopPoint(const double length, PathWithLaneId & path);
 double getSignedDistanceFromBoundary(
   const lanelet::ConstLanelets & shoulder_lanelets, const Pose & pose, const bool left_side);
 std::optional<double> getSignedDistanceFromBoundary(
```

Comment:
Is it also an option to take the `vehicle_info_util::VehicleInfo` type as an argument?
This member variable is necessary for safety check in goal_planner, so i added in this [PR](https://github.com/autowarefoundation/autoware.universe/pull/4859)

Reply:
yes it is also ok for me.

Reply:
merged in the [PR](https://github.com/autowarefoundation/autoware.universe/pull/4859)